src/main/java/tg/kindhands_bot/kindhands/components/ProcessingBotMessages.java

```diff
@@ -83,4 +139,34 @@ public SendMessage blockedMessage() {
         String answer = firstNameUser + ", ваш аккаунт заблокирован";
         return returnMessage(answer);
     }
+
+    /**
+     * Меняет статус бота для пользователя на NULL
+     * -----||-----
+     * Sending a message when user is blocked
+     */
+    public void changeStateBotNull() {
+        User user = userRepository.findByChatId(update.getMessage().getChatId());
+        if (user == null) throw new NullPointerException();
```

Comment:
```suggestion
        if (user == null) {
            throw new NullPointerException();
        }
```
скобки всегда надо писать